webui/react/src/App.tsx

```diff
@@ -46,15 +47,19 @@ const AppView: React.FC = () => {
     setInfo({ type: Info.ActionType.Set, value: infoResponse.data });
   }, [ infoResponse, setInfo ]);
 
+  // FIXME layout
```

Comment:
LGTM, anything else that needs fixing?

Reply:
I think this was just left here from a rebase conflict resolution. will remove